api.js

```diff
@@ -1194,11 +1184,6 @@ module.exports = async function attachAPI(app, {wss, db}) {
 
   app.get('/api/user-list', [
     ...middleware.runIfCondition(() => shouldUseAuthorization, [
-      ...middleware.loadVarFromQuery('sessionID', false),
-      ...middleware.runIfVarExists('sessionID',
-        middleware.getSessionUserFromID('sessionID', 'sessionUser')
-      ),
-
```

Comment:
This gets rid of the `sessionUser` middleware variable so it is probably broken.

Reply:
As far as I know all authenticated endpoints get the sessionUser variable for free.

Reply:
I mean, the only solution to this argument is to *test* `/api/user-list` (but I'm not quite awake yet). I don't see anything that's *added* to save `sessionUser`, though; only that it now saves <code>session<em>Id</em></code> for free.